Sources/JetStream/context.swift

```diff
@@ -145,3 +165,11 @@ struct Ack: Codable {
         duplicate = try container.decodeIfPresent(Bool.self, forKey: .duplicate) ?? false
     }
 }
+
+/// contains info about the `JetStream` usage from the current account.
+public struct AccountInfo: Codable {
```

Comment:
We can do it in a separate PR, but this is not a full account info response

Reply:
Yeah, I just wanted something that I can test against.

Let's make work on it when we will add the AccountInfo PR.